cmake/Dependencies.cmake

```diff
@@ -108,6 +108,18 @@ if(USE_NNPACK)
   endif()
 endif()
 
+# ---[ QNNPACK, use gemmlowp
+if(USE_GEMMLOWP)
```

Comment:
`USE_INT8_OPS` would be a better name as QNNPACK itself doesn't depend on gemmlowp